docker/Dockerfile.release

```diff
@@ -39,7 +39,7 @@ RUN export version=$(echo "${source_branch}" | cut -d '-' -f 2) && \
   (mkdir stackdriver && cd stackdriver && jar -xf ../zipkin-module-storage-stackdriver-${version}-module.jar) && \
   rm zipkin-module-storage-stackdriver-${version}-module.jar
 
-FROM openzipkin/zipkin:2.18.3
+FROM openzipkin/zipkin:2.19.1
```

Comment:
I wanted to derive this from maven version, but it seems people have already tried and failed at that... https://stackoverflow.com/questions/36556766/can-a-variable-be-used-in-docker-from